pkg/kudoctl/cmd/update.go

```diff
@@ -0,0 +1,108 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/cmd/install"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	updateExample = `
+		The update argument must be a name of the instance.
+
+		# Update dev-flink instance with setting parameter param with value value
+		kubectl kudo upgrade dev-flink -p param=value
+
+		# Update dev-flink instance in namespace services with setting parameter param with value value
+		kubectl kudo upgrade dev-flink -n services -p param=value`
+)
+
+type updateOptions struct {
+	Namespace  string
+	Parameters map[string]string
+}
+
+// defaultOptions initializes the install command options to its defaults
+var defaultUpdateOptions = &updateOptions{
+	Namespace: "default",
+}
+
+// newUpgradeCmd creates the install command for the CLI
+func newUpdateCmd() *cobra.Command {
+	options := defaultUpdateOptions
+	var parameters []string
+	upgradeCmd := &cobra.Command{
+		Use:     "update <instance-name>",
+		Short:   "Update installed KUDO operator.",
+		Long:    `Update installed KUDO operator with new parameters.`,
+		Example: updateExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Prior to command execution we parse and validate passed parameters
+			var err error
+			options.Parameters, err = install.GetParameterMap(parameters)
+			if err != nil {
+				return errors.WithMessage(err, "could not parse parameters")
+			}
+			return runUpdate(args, options)
+		},
+		SilenceUsage: true,
```

Comment:
Why are we setting this?

Reply:
lol good question :D I don't know. Copy-paste 😛 

`SilenceUsage is an option to silence usage when an error occurs.` ... sounds reasonable to me though 🤷‍♀ lot of errors can come up from non-usage errors so maybe that's why we have decided to not print it there? No hard opinion though...

Reply:
okay, should I remove it then from all comands?